src/pybind/mgr/selftest/module.py

```diff
@@ -204,6 +224,39 @@ def _self_test_osdmap(self):
 
         self.log.info("Finished self-test procedure.")
 
+    def _test_remote_calls(self):
+        # Test making valid call
+        self.remote("influx", "handle_command", "", {"prefix": "influx self-test"})
+
+        # Test calling module that exists but isn't enabled
+        mgr_map = self.get("mgr_map")
+        all_modules = [m['name'] for m in mgr_map['available_modules']]
+        disabled_modules = set(all_modules) - set(mgr_map['modules'])
+        disabled_module = list(disabled_modules)[0]
+        try:
+            self.remote(disabled_module, "handle_command", {"prefix": "influx self-test"})
```

Comment:
Why bother with the `"handle_command"` and `{"prefix": "influx self-test"}` arguments here? They won't be correct for a random disabled module. Same comment for the subsequent calls to test modules that don't exist and methods that don't exist.

Not that it actually makes any practical difference though, I guess; the various deliberate failures will occur regardless of the arguments passed.

Reply:
Just so that when I'm passing in an invalid module, the other arguments are at least somewhat valid-looking

Reply:
Fair enough.